src/_base/application/overlay/Jenkinsfile.twig

```diff
@@ -56,7 +56,9 @@ pipeline {
                 milestone(50)
                 sh 'ws app publish'
 {% if @('pipeline.publish.chart.enabled') %}
-                sh 'ws app publish chart "${GIT_BRANCH}" "{{ @('workspace.name') }} build artifact ${GIT_COMMIT}"'
+                lock(resource: '{{ @('workspace.name') }}-publish-chart', inversePrecedence: true) {
```

Comment:
since there are potentially multiple projects released to a cluster repository, would it be better to use the repository itself, otherwise it only prevents individual projects

Reply:
or, if we always keep the same folder structure in jenkins, the folder path

Reply:
Good point, will update :+1:

Reply:
Updated

Reply:
One more file!